resources/sass/homePage/home.scss

```diff
@@ -7,69 +7,58 @@ line-height: 26px;
 color: #C9C9C9;
 
 } */
-.home {
-   a{
-    width: 106px;
-    height: 22px;
-    color: #555982;
-    display: flex;
-    justify-content: space-around;
-    align-items: right;
-  }
-  a:hover{
-   color: #5F69D3;
-   }
-  a:active{
-   color: #4954C6;
-   }
-
-   div.Method_pap{
-    color: #646BAB;
-   /* display: flex; */
-   justify-content: space-around;
-   align-items: right;
-
-   }
-   button.test {
-     width: 285px;
-     height: 80px;
-     background: #FF3062;
-     color: #FFFFFF;
-  box-shadow: 0px 4px 20px rgba(255, 53, 53, 0.25);
-  border-radius: 50px;
-   }
-
-   button.more {
-     width: 166px;
-  height: 46.6px;
-  /* left: 882px;
-  top: 3403px; */
-  color: #FFFFFF;
-  background: #FF3062;
-  box-shadow: 0px 4px 20px rgba(255, 53, 53, 0.25);
-  border-radius: 50px;
-   }
-
-   button.more_articles {
-     width: 331px;
-   height: 50px;
-   /* left: 882px;
-   top: 3403px; */
-   align-items: center;
-   color: #FFFFFF;
-   background: #FF3062;
-   box-shadow: 0px 4px 20px rgba(255, 53, 53, 0.25);
-   border-radius: 50px;
-   }
-
-  div.era img{
-    width: 100%;
-    /* width: 628px;
-    height: 628px; */
-  }
-  div.stepall{
-    width: 1159px;
-  /* height: 659px; */
+body.home {
+  font-size: 18px;
+  background: url(/images/background.jpg);
+  /* background-attachment: fixed; */
+  background-size: 100%;
+}
+
+h1 {
```

Comment:
please not to setup styles for all elements
instead of set style for h1 add some class to h1 and set styles for this class